oshi-core/src/main/java/oshi/jna/platform/linux/LinuxLibc.java

```diff
@@ -67,10 +68,20 @@ class Ut_Tv extends Structure {
     LinuxUtmpx getutxent();
 
     /**
-     * Returns the caller's thread ID (TID). In a single-threaded process, the thread ID is equal to the process ID. In
-     * a multithreaded process, all threads have the same PID, but each one has a unique TID.
+     * SYS_gettid
+     * Defined in /usr/include/asm/unistd_32.h or /usr/include/asm/unistd_64.h
+     */
+    int SYS_GETTID = Platform.is64Bit() ? 186 : 224;
```

Comment:
So it looks like there are 3 options for this, so you'll need another conditional based on `Platform.isARM()`:
 - x86_64: 0xba = 186
 - x86 or arm: 0xe0 = 224
 - arm64: 0xb2 = 178